### DISCUSSION TITLE
Music Composition with Deep Learning

### DISCUSSION THREAD

**Discussion by Pujuu548** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#103] [General]
Hey all myself POOJA . I am student from UMIT MUMBAI  interested in working and collaborating on this project [Music Composition with Deep Learning]

**Comment by robertogl** [comment]
Hi @Pujuu548 , 

To enhance the organization of the repository, I have set up a discussion forum for each project. Therefore, kindly utilize the existing forum for this project. You can find the link below:
https://github.com/mathworks/MATLAB-Simulink-Challenge-Project-Hub/discussions/78
